src/SWP/Component/TemplatesSystem/Gimme/Meta/Meta.php

```diff
@@ -201,14 +201,19 @@ private function fillFromArray(array $values, array $configuration, string $name
     private function fillFromObject($values, array $configuration, string $name)
     {
         if (isset($configuration['properties'][$name])) {
+            $type = $configuration['properties'][$name]['type'];
+
             $getterName = 'get'.ucfirst($name);
+
+            if ('bool' === $type) {
+                $getterName = 'is'.ucfirst($name);
+            }
+
             if (method_exists($values, $getterName)) {
                 $this->$name = $values->$getterName();
             }
         }
 
-        unset($values, $configuration);
```

Comment:
whey remove this? It can save some memory especially on page where we load lot's of meta. As every meta is new instance.

Reply:
Static Code Analyzer says

```
Only local copy/reference will be unset. This unset can probably be removed.
Unset operations applied to callable parameters only destroy local copies/links. When parameters are not re-used, this operation can be safely removed
```